tools/internal_ci/linux/pull_request/grpc_interop_matrix_adhoc.cfg

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 gRPC authors.
```

Comment:
What's the purpose of this file?

Reply:
I think it is useful - it allows to run an interop_matrix job on any future pull request, which seems to be pretty useful feature to have - we from time make changes to run_interop_matrix_tests.py script and being able to test them easily seems valuable.